### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by BaniaFonseca**
Hi there

I have done minor changes on the project:

1) refacoted the readme
2) created download_models.sh
3) refactored Makefile

### CODE REVIEW

File: gym/download_models.bash [hunk 1]
```diff
@@ -0,0 +1,5 @@
+wget http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8.tar.gz
```
Comment (nlauchande) [inline gym/download_models.bash:1]: I wonder if it makes sense to make it a bash function where you specify the URI and target directory to deploy the model.

**Review by nlauchande — APPROVED** [review]
LGTM

File: gym/download_models.bash (hunk 1 above)
Comment (BaniaFonseca) [reply gym/download_models.bash:1]: what you meant by that?

is not model deployment, it is a pretrained model, 
since .gitignore do not allow  to push pretrained model
I wrote this script to make it easy for people to download the pre trained model

File: gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+all_model_checkpoint_paths: "ckpt-6"
+all_model_checkpoint_paths: "ckpt-7"
+all_model_checkpoint_timestamps: 1611132306.153513
+all_model_checkpoint_timestamps: 1611132921.8376176
+all_model_checkpoint_timestamps: 1611133507.366148
+all_model_checkpoint_timestamps: 1611134092.1264625
+all_model_checkpoint_timestamps: 1611134677.455968
+all_model_checkpoint_timestamps: 1611135262.102583
+all_model_checkpoint_timestamps: 1611135849.2558124
+last_preserved_timestamp: 1610998242.017333
```
Comment (tjiagoM) [inline gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint:16]: I'm not sure I understand what is this @BaniaFonseca. Is this supposed to be in the repo?

File: gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/pipeline.config [hunk 3]
```diff
@@ -138,10 +138,19 @@ train_config {
...
+ }
+ data_augmentation_options{
+  random_adjust_saturation{
+  }
+ }
+ data_augmentation_options{
+  random_adjust_brightness{
+  }
+ }
+   data_augmentation_options {
```
Comment (tjiagoM) [inline gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/pipeline.config:153]: We don't need to add this to this PR, but I believe it will be useful to add in the future `random_image_scale`, `random_distort_color`, `random_black_patches`, etc, or even use a few from https://github.com/aleju/imgaug

Comment (tjiagoM) [reply gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/pipeline.config:153]: (to clarify: this is just suggestions for the future)

File: gym/hostname.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (tjiagoM) [inline gym/hostname.sh:1]: Why is this file necessary?

File: gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint (hunk 2 above)
Comment (BaniaFonseca) [reply gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint:16]: No

**Review by tjiagoM — APPROVED** [review]
Left a few comments, but overall lgtm :)

File: gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/pipeline.config (hunk 3 above)
Comment (BaniaFonseca) [reply gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/pipeline.config:153]: got it

File: gym/hostname.sh (hunk 4 above)
Comment (BaniaFonseca) [reply gym/hostname.sh:1]: ignore it, I was testing  slurm

File: gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint (hunk 2 above)
Comment (tjiagoM) [reply gym/ai4coral/models/ssd_mobilenet_v2_fpnlite/checkpoint:16]: Ah, then you probably to need to run `git rm` on this file, and add `checkpoint` to gitignore. I think the gitignore only has the *folder* checkpoint, rather than the file.

File: gym/hostname.sh (hunk 4 above)
Comment (tjiagoM) [reply gym/hostname.sh:1]: Ah, got it. If you are gonna further use/edit this then it's ok, otherwise I think you can delete already to not clutter things. up to you.

File: gym/download_models.bash (hunk 1 above)
Comment (nlauchande) [reply gym/download_models.bash:1]: Not a big issue.  Just a style thing:
imagine if you could call a bash/cli just with 

"download_model 
 --uri http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8.tar.gz"

And behind the scenes would do all the unpacking and storing in the correct place.

Comment (BaniaFonseca) [reply gym/download_models.bash:1]: I see,
I've first considered to do what you are saying but I found it expensive \
devs are not supposed to know where the model will be downloaded from \
that's why I decided to hard code it that way \
I needed to hid the details as much as possible from devs \
so that the only thing they have to do is to use make
